include/qpdf/QPDFTokenizer.hh

```diff
@@ -107,6 +107,27 @@ class QPDFTokenizer
                 (this->type != tt_bad) && (this->type == rhs.type) &&
                 (this->value == rhs.value));
         }
+        bool
+        isInteger() const
+        {
+            return this->type == tt_integer;
+        }
+        bool
+        isWord() const
+        {
+            return this->type == tt_word;
+        }
+        bool
+        isWord(std::string const& value) const
+        {
+            return this->type == tt_word && this->value == value;
+        }
+        bool
+        isWord(std::string const& value1, std::string const& value2) const
```

Comment:
I like the change except not this overload of isWord. You can just say `isWord("A") || isWord("B")` and not worry about the slight amount of extra work.

Reply:
I'll take this out